docs/source-pytorch/common/precision_intermediate.rst

```diff
@@ -158,18 +161,71 @@ Under the hood, we use `transformer_engine.pytorch.fp8_autocast <https://docs.nv
 ----
 
 
-***************
-8-bit Optimizer
-***************
+*****************************
+Quantization via Bitsandbytes
+*****************************
+
+`bitsandbytes <https://github.com/TimDettmers/bitsandbytes>`__ (BNB) is a library that supports quantizing Linear weights.
+
+Both 4-bit (`paper reference <https://arxiv.org/abs/2305.14314v1>`__) and 8-bit (`paper reference <https://arxiv.org/abs/2110.02861>`__) quantization is supported.
+Specifically, we support the following modes:
+* nf4: Uses the normalized float 4-bit data type. This is recommended over "fp4" based on the paper's experimental results and theoretical analysis.
+* nf4-dq: "dq" stands for "Double Quantization" which reduces the average memory footprint by quantizing the quantization constants. In average, this amounts to about 0.37 bits per parameter (approximately 3 GB for a 65B model).
+* fp4: Uses regular float 4-bit data type.
+* fp4-dq: "dq" stands for "Double Quantization" which reduces the average memory footprint by quantizing the quantization constants. In average, this amounts to about 0.37 bits per parameter (approximately 3 GB for a 65B model).
+* int8: Uses unsigned int8 data type.
+* int8-training: Meant for int8 activations with fp16 precision weights.
+
+While these techniques store weights in 4 or 8 bit, the computation still happens in 16 or 32-bit (float16, bfloat16, float32).
+This is configurable via the dtype argument in the plugin.
+
+Quantizing the model will dramatically reduce the weight's memory requirements but might impact performance negatively.
```

Comment:
Here, what you mean is performance in terms of predictive performance (e.g. accuracy of the classifier), not runtime performance. This could be clarified.